src/show/components/Navbar/index.js

```diff
@@ -107,25 +132,56 @@ const Navbar = () => {
             <form className='mt-8 space-y-6'>
               <div className='rounded-md shadow-sm -space-y-px'>
                 <TextField
+                  onChange={e =>
+                    setSignUpUser({ ...signUpUser, username: e.target.value })
```

Comment:
Seems like you are using this quite a lot so you can create a function that accepts two params, the key to be added/edited in the `signupUser` object and the second one the value.

Reply:
gata sefu, 3 ore am muncit